tools/make/tests.toml

```diff
@@ -34,6 +34,7 @@ description = "Run all Rust tests with all features and targets"
 category = "ICU4X Development"
 # This might be required if Windows runs out of disk again
 # env = { RUSTFLAGS = "-Cdebuginfo=none -Cdebug-assertions=true", condition = { platforms = ["windows"] } }
+env = { RUSTFLAGS = "--cfg=icu4x_run_size_tests=\"1\"" }
```

Comment:
You don't need an = here either. It's not an environment variable.

Reply:
In fact, doing it this way _won't work_, `cfg(icu4x_run_size_tests)` is distinct from `cfg(icu4x_run_size_tests = 1)`. The latter does not enable the former.

Custom CFGs are simple flags with no additional magic, they just have a grammar to themselves to allow people to build up complex CFGs, but they still just behave as flags.